test/patterns/redundant_catch/test_redundant_catch.py

```diff
@@ -104,4 +104,4 @@ def test_fake_try_in_lambda(self):
         """
         pattern = RedundantCatch()
         lines = pattern.value(os.path.dirname(os.path.realpath(__file__)) + '/Cache.java')
-        self.assertEqual(lines, [])
+        self.assertEqual(lines, [393])
```

Comment:
no, it's wrong behaviour. W have lambda. There is no another way to handle exception in lambda. There was a test to check that. If we have try in lambda, we should ignore it

Reply:
@lyriccoder  I FIXED IT!